### ISSUE TITLE
Better training results

### ISSUE THREAD

**Issue by Egrt** [Xunius/glasses_CycleGAN#3]
I have used a modified CyclGAN based on **SwinIR** which can have better training results.

1. SwinTransformer was used as the generative network backbone.
2. RWGAN-GP is used as GAN loss.
3. Introduced ResNest face recognition network supervised model to repair faces without losing original identity features.
4. Perceptual loss function can also be introduced (with possible enhancements).

Here are my results after training 10 epochs:

![image](https://user-images.githubusercontent.com/47631482/195527320-7aa7e770-75ac-40bf-85a0-58d1c19118d9.png)
